identity-service/src/notifications/fetchNotificationMetadata.js

```diff
@@ -273,9 +273,11 @@ async function fetchNotificationMetadata (audius, userId, notifications, fetchTh
     return um
   }, {})
 
-  for (let trackId of Object.keys(trackMap)) {
-    const track = trackMap[trackId]
-    track.thumbnail = await getTrackImage(track, userMap)
+  if (fetchThumbnails) {
+    for (let trackId of Object.keys(trackMap)) {
```

Comment:
does getTrackImage set the thumbnail inside userMap? i guess i don't see where the const track object is referenced later...? bc we return trackMap, collectionMap, userMap right after this so one of them must be getting modified?

Reply:
this const is a reference to the entity _inside_ trackMap, we are modifying field - totally safe